main.py

```diff
@@ -1,10 +1,43 @@
 import init_django_orm  # noqa: F401
+import json
 
 from db.models import Race, Skill, Player, Guild
+from django.db.models import QuerySet
+
+
+def race_create(player_race_data: dict) -> QuerySet:
+    race_name = player_race_data["name"]
+    if not Race.objects.filter(name=race_name).exists():
+        race = Race.objects.create(name=race_name,
+                                   description=player_race_data["description"])
+        for skill in player_race_data["skills"]:
+            Skill.objects.create(name=skill["name"],
+                                 bonus=skill["bonus"],
+                                 race=race)
+    return Race.objects.get(name=race_name)
+
+
+def guild_create(guild_data: dict) -> QuerySet | None:
```

Comment:
for guild_data use typehint like: Optional[YOUR_VALUE_TYPE] = None
for example: list that contains string elements and can be None has typehint Optional[list[str]]